osbrain/tests/test_agent.py

```diff
@@ -301,7 +306,7 @@ def on_init(self):
     puller.bind('PULL', alias='pull', handler=append_received,
                 addr=address.address, transport='tcp')
 
-    assert should_receive == wait_agent_attr(puller, data='foo', timeout=.2)
+    assert should_receive == wait_agent_attr(puller, data='foo', timeout=1)
```

Comment:
I guess this timeout does not need to be changed for now as this test is skipped for Windows. When/if we remove the skip we will change the timeout if necessary.

Reply:
But this one does fail on Linux from time to time. I ran it in a loop yesterday, and it took me anywhere between 50 and 500 attempts to make it fail, but eventually it does.

It has failed a few times in travis:
https://travis-ci.org/ocaballeror/osbrain/jobs/346269407

I think 1 is a safer value.

Reply:
@ocaballeror Then ok. It seems changing the timeout does not actually change the test, will only make it a bit slower for the cases where `should_receive == False`, so it is fine for me. :blush:

Should we put it as a separate commit? i.e.: "More lenient timeout in linger test". Simply because it is not really related to Windows tests.

Reply:
Makes sense 👍 

Rebasing this is going to be fun....